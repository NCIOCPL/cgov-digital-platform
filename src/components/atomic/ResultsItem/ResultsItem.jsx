import React from 'react';
import PropTypes from 'prop-types';
import './ResultsItem.scss';

const ResultsItem = ({
  sampleProperty
}) => {
  return (
    <div className="results-item">
      { sampleProperty }
    </div>
  );
};

ResultsItem.propTypes = {
  sampleProperty: PropTypes.string
};

ResultsItem.defaultProps = {
  sampleProperty: 'ResultsItem'
};

export default ResultsItem;
