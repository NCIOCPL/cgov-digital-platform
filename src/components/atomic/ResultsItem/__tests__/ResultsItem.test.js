import React from 'react';
import { render, shallow, mount } from 'enzyme';
import {
  resetDom,
  setupDom
} from '../../../../utilities/testHelpers';
import ResultsItem from '../ResultsItem';

const defaultProps = {
}

const setup = (enzymeMethod = render, props = {}) => {
  const component = enzymeMethod(
    <ResultsItem {...props} />
  );
  return component;
};

describe('ResultsItem', () => {
  describe('Render', () => {
    let component = null;

    afterEach(() => {
      resetDom();
    });

    it('renders without error', () => {
      component = setup(render, defaultProps);
      expect(component).toMatchSnapshot();
    });
  });

});
