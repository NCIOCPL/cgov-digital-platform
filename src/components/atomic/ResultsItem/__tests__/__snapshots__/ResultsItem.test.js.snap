// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsItem Render renders without error 1`] = `
<div
  class="results-item"
>
  ResultsItem
</div>
`;
